docs/source/docking_requirements.rst

```diff
@@ -40,33 +40,34 @@ The Python package ``meeko`` is a new type of package developped in the Forli la
     - Hydrated docking
     - Macrocycles
 
-This new tool can be easily installed with all its depencies using `pip` or `conda` package managers.
+**Meeko installation using pip**:
 
-**Meeko installation using Conda**:
+.. note::
 
-The `Conda package manager <https://docs.conda.io/en/latest/>`_ is included as part of the Anaconda Python distribution, which can be download from `https://docs.continuum.io/anaconda/install <https://docs.continuum.io/anaconda/install/>`_. This is a Python distribution specially designed for scientific applications, with many of the most popular scientific packages preinstalled. Alternatively, you can use `Miniconda <https://conda.pydata.org/miniconda.html>`_, which includes only Python itself, plus the Conda package manager.
+    When using ``pip``, it's good pratice to use a virtual environment and also the easiest solution. An example with the `Conda package manager <https://docs.conda.io/en/latest/>`_ is available further down.
 
-1. Begin by installing the most recent 64 bit, Python 3.x version of either Anaconda or Miniconda
-2. (Optional, but highly suggested) If you want, you can create a dedicated environment for ``meeko`` and ``AutoDock Vina`` packages (see :ref:`installation`):
+.. warning::
+    
+    OpenBabel executable and library must be installed first, see `installation instruction <https://open-babel.readthedocs.io/en/latest/Installation/install.html#install-binaries>`_.
 
 .. code-block:: bash
+    
+    $ pip install -U numpy openbabel meeko
 
-    conda create -n vina python=3
-    conda activate vina
-    conda install -c conda-forge -c ccsb-scripps vina
-
-3. And type the following command
-
-.. code-block:: bash
+If the installation was successful, you should now be able to access to the following command from your terminal by typing:
 
-    conda install -c ccsb-scripps meeko
+    - mk_prepare_ligand.py
 
-**Meeko installation using pip**:
+**Meeko installation in a Conda environment**:
 
-.. code-block:: bash
+.. note::
 
-    pip install meeko
+    See instructions in :ref:`installation` on how to setup and create an dedicated ``Conda`` environment.
 
-If the installation was successful, you should now be able to access to the following command from your terminal by typing:
+Type the following command to install ``NumPy``, ``OpenBabel`` and ``meeko``:
 
-    - mk_prepare_ligand.py
+.. code-block:: bash
+    
+    $ conda activate vina
+    $ conda install numpy openbabel
```

Comment:
Seems like `openbabel` is at the conda-forge channel. I got `PackagesNotFoundError` unless `-c conda-forge` was specified.